website/www/site/content/en/contribute/_index.md

```diff
@@ -98,27 +111,44 @@ sudo apt-get install \
    docker-ce
 ```
 
-On some systems (like Ubuntu 20.04) these need to be installed also
+2. On some systems (like Ubuntu 20.04) these need to be installed also
+
 ```
 pip3 install grpcio-tools mypy-protobuf
 ```
 
-You also need to [install Go](https://golang.org/doc/install).
-
-Once Go is installed, install goavro:
+3. If you you develop in GO:
+    1. Install [Go](https://golang.org/doc/install).
+    1. Check BEAM repo is in: `$GOPATH/src/github.com/apache/`
+    1. At the end, it should look like this: `$GOPATH/src/github.com/apache/beam`
+4. Once Go is installed, install goavro:
 
 ```
 $ export GOPATH=`pwd`/sdks/go/examples/.gogradle/project_gopath
 $ go get github.com/linkedin/goavro
 ```
 
-gLinux users should configure their machines for sudoless Docker.
+**Important**: gLinux users should configure their machines for sudoless Docker.
+
+##### Automated script for Linux and macOS
+
+You can install these in a Debian-based distribution for Linux or macOs using the [local-env-setup.sh](https://github.com/apache/beam/blob/master/local-env-setup.sh)
+script, which is part of the Beam repo:
```

Comment:
I think it would help here if you would please describe which tools the script installs.
Or mention that the script will print a message or something about which tools its installing.

Reply:
I included this suggestion in the new commit.